b2/bucket.py

```diff
@@ -370,6 +373,7 @@ def _upload_small_file(
                     return FileVersionInfoFactory.from_api_response(upload_response)
 
             except B2Error as e:
+                logger.exception('error when uploading, token was %s', upload_auth_token)
```

Comment:
Don't log the auth token.  That *is* sensitive.  It's the URL that's interesting for finding the logs on the server side.